core/src/main/res/values/strings.xml

```diff
@@ -197,6 +209,8 @@
     <string name="remove_new_flag_label">Remove \"new\" flag</string>
     <string name="removed_new_flag_label">Removed \"new\" flag</string>
     <string name="mark_read_label">Mark as played</string>
+    <string name="marked_read_label">Marked as played</string>
```

Comment:
I wouldn't display a snackbar here. We already show way too many of them and they are pretty distracting.

Reply:
hmm i thought it is usefull, i can remove it though...

Reply:
Well yea it surely could be down but I'm just not convinced that it is worth it, and I'm not sure if I want to tinker with this. Either way I guess we could just add the basic functionality with this PR and optimize it later on (if that's something you wanna do) cause I actually only really care for the functionality 😅

Reply:
Maybe we can establish if there is a real performance impact (I'd argue not worse than the context menu actions) or if this is "just" a UI/UX enhancement for which I'm not sure it's worth it to add all this logic to the SwipeActions.

Reply:
Also there are other possible combinations other than playstate, like downloaded or favorite and maybe more if other SwipeActions or filters are added (see here for the request of even more finegrained filters https://forum.antennapod.org/t/organising-new-podcast-epsiodes/1031/4)